localization/react-intl/src/app/components/drawer/Projects/ArticlesComponent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "id": "articlesComponent.ArticlesNavHeader",
+    "description": "The navigation name of the articles section",
+    "defaultMessage": "Articles"
+  },
+  {
+    "id": "articlesComponent.claim&factChecks",
+    "description": "Label for a list displayed on the left sidebar that includes items that have claim and fact-checks",
+    "defaultMessage": "Claim & Fact-Checks"
+  },
+  {
+    "id": "articlesComponent.explainers",
+    "description": "Label for a list displayed on the left sidebar that includes items that have explainers",
+    "defaultMessage": "Explainers"
+  },
+  {
+    "id": "articlesComponent.importedReports",
+    "description": "Label for a list displayed on the left sidebar that includes items from the 'Imported fact-checks' channel",
+    "defaultMessage": "Imported"
+  },
+  {
+    "id": "articlesComponent.published",
+    "description": "Label for a list displayed on the left sidebar that includes items that have published reports",
+    "defaultMessage": "Published"
+  }
+]
```

Comment:
These can keep their original ids `projectsComponent.importedReports` and `projectsComponent.publish` so they don't need to be retranslated